lib/sycamore/sycamore/data/table.py

```diff
@@ -0,0 +1,164 @@
+from collections import OrderedDict
+from dataclasses import dataclass, field
+from typing import Any, Optional
+import xml.etree.ElementTree as ET
+
+from sycamore.data import BoundingBox
+from PIL import Image, ImageDraw
+import numpy as np
+from pandas import DataFrame
+
+
+# This data model is similar to that used by Textract and TableTransformers.
+# It is intended to be a common intermediate representation for a variety of
+# table structure recognition models.
+@dataclass(frozen=True)
+class TableCell:
+    content: str
```

Comment:
Any support for nested tables?

Reply:
No, not right now. Honestly I didn't think about it because none of the detection models I know of support them. I would suggest we defer this until it comes up.

Reply:
should we just call this text_representation as well? Unless it has a different meaning

Reply:
Debating this. In hindsight I wish we hadn't called it text_representation for Documents, and now I'm not sure whether to go for consistency or clarity.